learning/modules/quantum-mechanics/stern-gerlach-measurements-with-qiskit.ipynb

```diff
@@ -1410,14 +1423,16 @@
    "id": "fbf48a0c-beaa-4ebe-a88c-213b940cf079",
    "metadata": {},
    "source": [
-    "## Acknowledgements\n",
+    "### Acknowledgements\n",
```

Comment:
```suggestion
    "### Acknowledgments\n",
```

Reply:
(IBM Style actually calls this one out in the style guide, not to use that middle e!)